index.html

```diff
@@ -64,25 +64,45 @@
     </section>
 
     <!-- About -->
-    <section id="about" class="section-pad-top">
-
-        <div id="about-content" class="section-pad">
-            <div id="about-text">
-                <h2>What is a Hackathon?</h2>
-                <p>A Hackathon is an invention marathon. Any student can attend! Really, any student... and it's completely free.</p>
-                <p>Students work in teams and mentors are present to offer help. Sponsors attend to help, recruit, and promote their products. Teams compete in different categories and prizes are awarded, but everyone learns and everyone wins!</p>
-                <p>Come spend 24 hours with us at BrickHack and dedicate time to learn, collaborate, build, and innovate.</p>
+    <section id="about">
+        <div id="about-text">
+            <h2>What is a Hackathon?</h2>
+            <p>A hackathon is an invention marathon. BrickHack is a 24-hour digital hackathon that is completely free and open to any university student 18+ years old.</p>
+            <p>Apply to come spend March 5th and 6th, 2022 with us to learn, collaborate, build, and innovate!</p>
+        </div>
+        <div id="about-cards">
+            <div class="card">
+                <h3>Applying</h3>
+                <p>When applications open, you will apply to BrickHack via the HackEngine system. Applications will be open until the event opens on March 5th.</p>
+                <p>If your application is accepted, you will receive an email from our team with more information.</p>
+                <p>You must be 18+ and a current university student to apply. Applications and event attendance are completely free.</p>
             </div>
-            <div id="about-buttons">
-                <a href="https://mlh.io/" target="_blank" rel="noopener noreferrer">
-                    <i class="fas fa-arrow-right"></i>
-                    Visit Major League Hacking
-                </a>
-                <a href="./gallery.html" rel="noopener noreferrer">
-                    <i class="fas fa-images"></i>
-                    BrickHack Gallery
-                </a>
+            <div class="card">
+                <h3>Participating</h3>
+                <p>At BrickHack, students compete individually or in teams of 2 - 4 to build anything they want. You could make an app, a website, a game, a design, or any other form of interactive media. No experience is required; we provide mentors to help you along the way!</p>
```

Comment:
content nit: Maybe get rid of the spaces between the numbers and hyphens for "2 - 4"? I usually see it without spaces.
```suggestion
                <p>At BrickHack, students compete individually or in teams of 2-4 to build anything they want. You could make an app, a website, a game, a design, or any other form of interactive media. No experience is required; we provide mentors to help you along the way!</p>
```

Reply:
Personal writing style choice. Fixing it